Core/HLE/sceKernelModule.cpp

```diff
@@ -810,10 +810,12 @@ void ImportFuncSymbol(const FuncSymbolImport &func, bool reimporting, const char
 }
 
 void ExportFuncSymbol(const FuncSymbolExport &func) {
-	if (FuncImportIsSyscall(func.moduleName, func.nid)) {
-		// HLE covers this already - let's ignore the function.
-		WARN_LOG(LOADER, "Ignoring func export %s/%08x, already implemented in HLE.", func.moduleName, func.nid);
-		return;
+	if (strcmp(func.moduleName, "sceLibfont") == 0) {
+		if (FuncImportIsSyscall(func.moduleName, func.nid)) {
```

Comment:
Can you explain this logic? Seems to me that now, FuncImportIsSyscall will only be checked if func.moduleName == "sceLibFont". Couldn't this change behavior or break things for other modules? I don't really understand.